en/contribute/git_examples.md

```diff
@@ -72,30 +85,77 @@ For a good commit message, please refer to [Contributing](../contribute/README.m
   Responsible members of PX4 will now have a look at your contribution and decide if they want to integrate it.
   Check if they have questions on your changes every once in a while.
 
+
 ## Get a Specific Release
 
-To get the source code for a *specific older release*:
-* Clone the PX4-Autopilot repo and navigate into PX4-Autopilot directory:
+Specific PX4 point releases are made as tags of the [release branches](#get-a-release-branch), and are named using the format `v<release>`.
+These are [listed on Github here](https://github.com/PX4/PX4-Autopilot/releases?q=release&expanded=true) (or you can query all tags using `git tag -l`).
+
+To get the source code for a *specific older release* (tag):
+
+1. Clone the PX4-Autopilot repo and navigate into PX4-Autopilot directory:
+
+   ```sh
+   git clone https://github.com/PX4/PX4-Autopilot.git
+   cd PX4-Autopilot
+   ```
+
+1. Checkout code for particular tag (e.g. for tag v1.13.0-beta2)
+
+   ```sh
+   git checkout v1.13.0-beta2
+   ```
+
+1. Then update submodules as shown:
+
+   ```sh
+   git submodule init
+   git submodule update --recursive
+   ```
+
+
+## Get a Release Branch
+
+Releases branches are branched of `main`, and used to backport necessary changes from main into a release.
+The branches are named using the format `release/<release_number>` (e.g. `release/v1.13`).
+The are [listed here](https://github.com/PX4/PX4-Autopilot/branches/all?query=release).
+
+To get a release branch:
+
+* Clone the PX4-Autopilot repo and navigate into _PX4-Autopilot_ directory:
+
   ```sh
   git clone https://github.com/PX4/PX4-Autopilot.git
   cd PX4-Autopilot
   ```
-* List all releases (tags)
+  
+* Fetch the desired release branch.
+  For example, assuming you want the source for PX4 v1.13:
+
   ```sh
-  git tag -l
+  git fetch origin release/1.13
   ```
-* Checkout code for particular tag (e.g. for tag 1.7.4beta)
+
+* Checkout the code for the branch
+
   ```sh
-  git checkout v1.7.4beta
+  git checkout release/1.13
   ```
 
+* You will need to update submodules:
+
+  ```sh
+  git submodule init
+  git submodule update --recursive
+  ```
```

Comment:
Good advice. Things that make me pause:
- This topic is about git
- The make command will fail if you haven't already installed dependencies. (though I guess you should have)

I'm still very much inclined because as you say, switching branches is frustrating.  I think maybe I need to put much of this in the next section "Update submodule" or have a new section "Switching branches". Or both.

Reply:
> The make command will fail if you haven't already installed dependencies

I'm fairly sure that it won't fail even if the dependencies are not installed. Those make commands only call for git commands (or they silently wipe everything if they fail).
https://github.com/PX4/PX4-Autopilot/blob/main/Makefile#L496-L520

Reply:
`make` isn't installed on computers by default (at least Windows).

Reply:
Oh
I did not consider `make` itself to be the issue.

Reply:
No worries. It's a bit "overthinking it" since if you're on this page you already probably have your build environment set up.

SHould be good for your final review now.